R/utilities.R

```diff
@@ -4275,41 +4277,6 @@ classify_fl <- function(
             )
     }
 
-    ssm_available <- colnames(ssm_matrix)
-    ssm_requested <- c(
-        ssm_features,
-        hotspot_features
-    )
-    ssm_missing <- setdiff(
-        ssm_requested,
-        ssm_available
-    )
-
-    if(length(ssm_missing)>0){
-        message(
-            "ATTENTION: Not all features are available in maf data!"
-        )
-        message(
-            paste0(
-                "A total of ",
-                length(
-                   ssm_missing
-                ),
-                " features are missing:"
-            )
-        )
-        message(
-            paste(
-                ssm_missing,
-                collapse=", "
-            )
-        )
-        message(
-            "They will be set to 0, which may affect model performance."
-        )
-        ssm_matrix[,ssm_missing] <- 0
-    }
-
     # Generate binary matrix for ashm
     ashm_matrix <- get_ashm_count_matrix(
         ashm_features_bed,
```

Comment:
What is being used for this bed file? Seems like a specific file is needed but where does it come from?

Reply:
It is defined a bit more up, at the line 4184:
```
    ashm_features_bed <- grch37_ashm_regions %>%
        dplyr::mutate(
            name = gsub("-", "_", grch37_ashm_regions$name)
        ) %>%
        dplyr::filter(name %in% ashm_features | name %in% c("PAX5_TSS_1", "SGK1_TSS_1"))
```